mautic/mautic
# Fix multiselect custom field in campaign condition

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/5537
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

Bugfix/feature donate by www.newcom4u.ch

[//]: # ( Required: )
#### Description:

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create a multi-liste custom field
2. Create contact and add all values from multiselect
3. Create campagin with lead field condition and multiselect 
4. Action after condition was not trigger

#### Steps to test this PR:
1.  Repeat all steps
2.  See If camapign condition work properly
